env/env.go

```diff
@@ -31,43 +24,41 @@ func GetAsString(variableName string, defaultValue string) string {
 	return value
 }
 
-// GetAsFloat returns float64 value for given environment variable,
-// returns default if not found or couldn't be parsed.
+// GetAsFloat returns value for given environment variable, with default if not found
 func GetAsFloat(variableName string, defaultValue float64) (floatValue float64) {
 	stringValue := os.Getenv(variableName)
 	if stringValue == "" {
 		return defaultValue
 	}
-
 	var err error
 	if floatValue, err = strconv.ParseFloat(stringValue, 64); err != nil {
-		log.
-			WithField("variable", variableName).
-			WithField("value", stringValue).
-			WithField("error", err).
-			Error("Failed to parse string to float")
-		return defaultValue
+		log.Panicf("Failed to parse string %s to float - %+v", stringValue, err)
 	}
 	return
 }
 
-// GetAsInt returns int value for given environment variable,
-// returns default if not found or couldn't be parsed.
+// GetAsInt returns value for given environment variable, with default if not found
 func GetAsInt(variableName string, defaultValue int) (intValue int) {
 	stringValue := os.Getenv(variableName)
 	if stringValue == "" {
 		return defaultValue
 	}
-
 	var err error
 	if intValue, err = strconv.Atoi(stringValue); err != nil {
-		log.
-			WithField("variable", variableName).
-			WithField("value", stringValue).
-			WithField("error", err).
-			Error("Failed to parse string to int")
-		return defaultValue
+		log.Panicf("Failed to parse string %s to int - %+v", stringValue, err)
 	}
+	return
+}
 
+// GetAsBool returns value for given environment variable, with default if not found
+func GetAsBool(variableName string, defaultValue bool) (boolValue bool) {
```

Comment:
Wouldn't it be useful if this returned `(bool, error)`? That way we can signal to the caller if `strconv.ParseBool()` fails

Reply:
Good point, it might. Maybe good to add both? UnsafeGetAsBool and GetAsBool later on?
I think it is better to have panic than throw away the value when its invalid and returning the default.